src/common/components/select-repository-list/index.js

```diff
@@ -130,18 +130,29 @@ export class SelectRepositoryListComponent extends Component {
   renderRepoAmount() {
     const { ids, isFetching } = this.props.repositories;
     const { selectedRepositories } = this.props;
+    const { filteredRepos } = this.props;
+    let filteredAmount = null;
 
     // Return nothing until isFetching completes
     if (isFetching || ids.length === 0) {
       return;
     }
 
+    if (filteredRepos.length < ids.length && filteredRepos.length > 1) {
+      filteredAmount = <span>{ filteredRepos.length } matches in </span>;
+    } else if (filteredRepos.length < ids.length && filteredRepos.length === 1) {
+      filteredAmount = <span>{ filteredRepos.length } match in </span>;
+    } else if (filteredRepos.length === 0) {
+      filteredAmount = <span>no matches </span>;
```

Comment:
"no matches" needs to be in sentence case if there are no selected repos: "No matches in 42 repos".